File: DSA With C/queue.c
#include<stdio.h>
#include<conio.h>
#include<stdlib.h>
#define SIZE 3
void enqueue();
void dequeue();
void display();
int c;
int rear=-1,front=-1,queue[SIZE];
int main()
{
 for(;;)
 {
  printf("******MENU******");
  printf("\n 1.For Enqueue\n 2.For Dequeue\n 3.For exit\n\n");
  printf("Enter the choice=");
  scanf("%d",&c);
  switch(c)
  {
    case 1:
      enqueue();
      break;
    case 2:
      dequeue();
      break;
    case 3:
      exit(1);
    default:
      printf("VAAIIIII!!!!! VALOIII BOLLL VUL VAL CHOICE DICHISSS.....\n TOR E TO SOMOY!!!!\n");
  };
 }
  getch();
}
void enqueue()
{
  if (rear==SIZE-1)
  {
      printf("QUEUE Over flow!!!!!!!!!!!!!!!!\n");
      return; 
  }
  else
  {
    if(front==-1)
       front=0;
    printf("ENter the Element Youn want to insert=");
    scanf("%d",&queue[++rear]); 
  }
  display();
}
void display()
{
  if (front==-1 || front>rear)
    {
        printf("Queue  underflow..... Mne vaiii queue faka ... kichu dekha r neii...\n");
        return;
    }
  else
  {
    int i;
    for(i=front;i<=rear;i++)
    {
      printf("%d\n",queue[i]);
    }
    printf("\n");
  }
}
void dequeue()
{
  if(front==-1)
  {
    printf("Queue Underflow!!!!!!\n");
    return;
  }
  else
  {
    printf("Deleted %d \n",queue[front]);
    front+=1;
  }
  display();
}

File: DSA With Java/Code/using Array/Queue.java
import java.util.*;
class B
{ Scanner kb=new Scanner(System.in);
  int f=-1,r=-1,n=5,queue[]=new int[n];
  
  public void enqueue()
  {
   if(r==n-1)
   {
     System.out.println("Queue is full !!");
     return;
   }
   if(f==-1)
     f=0;
   System.out.println("Enter your Element = ");
   int a=kb.nextInt();
   System.out.println("Insertion success");
   queue[++r]=a;
   peek();
  }
  
  public void dequeue()
  {
   if(f==-1 || f>r)
   {
    System.out.println("Queue is MT :( ");
    return;
   }
   System.out.println("Deleted  "+queue[f++]);
   peek();
  }
  public void peek()
  {
    if(f==-1 || f>r)
    {
        System.out.println("Q is MT :(");
        return;
    }
    for(int i=f;i<=r;i++)
    {
      System.out.print(" "+queue[i]);
    }
  }
}

class Queue
{
    public static void main(String[] args )
    {   B cv =new B();
        int ch;
        while(true)
        {
            Scanner ob = new Scanner(System.in);
            System.out.print("\n1. For Enqueue \n2. For Dequeue \n3. For exit");
            System.out.println("\nEnter your choice = ");
            ch=ob.nextInt();
            switch(ch)
            {   
                case 1:
                    cv.enqueue();
                    break;
                case 2:
                    cv.dequeue();
                    break;
                case 3:
                     System.exit(0);
                default:
                     System.out.println("Areeeeee vaii vul choiceee");
            }
        }
    }
}